stone/backends/python_types.py

```diff
@@ -555,6 +642,110 @@ def _generate_struct_class_properties(self, ns, data_type):
                 self.emit('self._{}_present = False'.format(field_name))
             self.emit()
 
+    def _generate_custom_annotation_instance(self, ns, annotation):
+        """
+        Generates code to construct an instance of an annotation type object
+        with parameters from the specified annotation.
+        """
+        annotation_class = class_name_for_annotation_type(annotation.annotation_type, ns)
+        return generate_func_call(
+            annotation_class,
+            kwargs=((fmt_var(k, True), self._generate_python_value(ns, v))
+                    for k, v in annotation.kwargs.items())
+        )
+
+    def _generate_custom_annotation_processors(self, ns, data_type):
```

Comment:
We'll likely want to do some refactoring into a helper function when implementing for non-Python backends to avoid reimplementing this tricky part, but looks OK for now